apps/src/code-studio/progressRedux.js

```diff
@@ -298,21 +298,21 @@ export const getLevelResult = level => {
 };
 
 /**
- * Does some processing of our passed in stages, namely
+ * Does some processing of our passed in lesson, namely
  * - Removes 'hidden' field
- * - Adds 'stageNumber' field for non-lockable, non-PLC stages
+ * - Adds 'stageNumber' field for non-lockable lessons or lessons with lesson plans, non-PLC lessons
```

Comment:
does this mean (lessons or lessons with lesson plans) which are also non-PLC ? can you clarify?